common/pop_types/cwp_investors.txt

```diff
@@ -46,6 +46,8 @@
 			}
 			if = {
 				limit = {
+					is_pop_type = engineers
```

Comment:
I agree with professionals here, not so much with engineers for reasons stated above.

Reply:
Well landlords used to be executives, and investors were meant to be more luxurious version of it.
Now its just rural/urban divide of upper class ownership.
So instead engineers shouldn't be more likely to become investors as well.